src/vm/instruction_set.md

```diff
@@ -1224,7 +1224,7 @@ Panic if:
 - `$rA + 32` overflows
 - `$ssp + $rC > VM_MAX_RAM`
 - `$rA + 32 > VM_MAX_RAM`
-- `$ssp + $rC > $hp - 1`
+- `$ssp + $rC > $hp`
```

Comment:
if `$hp` is really the start of the addressable heap, then we shouldn't allow `$sp == $hp`.

```suggestion
- `$ssp + $rC >= $hp`
```